khmer/_oxli/graphs.pxd

```diff
@@ -42,16 +42,16 @@ cdef extern from "oxli/hashtable.hh" namespace "oxli":
         void count(const char *)
         void count(HashIntoType)
         bool add(const char *)
-        bool add(HashIntoType)
+        #bool add(HashIntoType)
```

Comment:
A "hack" found on SO: change the second line to `bool add2 "add"(HashIntoType)`. Somehow tricks cython but because you provide a "real name" that is the same as above the compiler figures it all out. This seems to make the compiler happy but segfaults the second time I try to add a kmer :-/